example_code/applications/vending_machine/.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## Description
```

Comment:
This file isn't necessary right?

Reply:
yea I deleted the folder
didn't notice that, thanks for the flag!